src/unit_tests/wazuh_modules/vulnerability_detector/test_wm_vuln_detector.c

```diff
@@ -152,9 +183,186 @@ const char *__wrap_sqlite3_errmsg(){
     return mock_ptr_type(const char *);
 }
 
-
 /* tests */
 
+void test_pkg_version_relate_epoch(void **state)
+{
+    struct pkg_version *version_a = state[0];
+    struct pkg_version *version_b = state[1];
+
+    version_a->epoch = 1;
+    version_a->version = "";
+    version_a->revision = "";
+    version_b->epoch = 0;
+    version_b->version = "";
+    version_b->revision = "";
+
+    int ret = pkg_version_relate(version_a, PKG_RELATION_NONE, version_b, VER_TYPE_NONE);
+    assert_int_equal(ret, 1);
+    ret = pkg_version_relate(version_a, PKG_RELATION_GT, version_b, VER_TYPE_DEB);
```

Comment:
For cases like this one, you would create different tests to make sure you are testing only one particular situation.

Reply:
So, do you mean that I should create a different test case for each `PKG_RELATION_XXX` where `version_a->epoch = 1` and `version_b->epoch = 0` ??

Reply:
DONE